geode-core/src/distributedTest/java/org/apache/geode/management/internal/cli/commands/RepeatedRebalanceDUnitTest.java

```diff
@@ -0,0 +1,216 @@
+            "Total time (in milliseconds) spent switching the primary state of buckets during this rebalance | 0")
+        .doesNotContainOutput(
+            "Total primaries transferred during this rebalance                                               | 0");
+    System.out.println("DEBR Finished rebalance one");
+
+    gfsh.executeAndAssertThat("rebalance").containsOutput(
+        "Total bytes in all redundant bucket copies created during this rebalance                        | 0")
+        .containsOutput(
+            "Total time (in milliseconds) spent creating redundant bucket copies during this rebalance       | 0")
+        .containsOutput(
+            "Total number of redundant copies created during this rebalance                                  | 0")
+        .containsOutput(
+            "Total bytes in buckets moved during this rebalance                                              | 0")
+        .containsOutput(
+            "Total time (in milliseconds) spent moving buckets during this rebalance                         | 0")
+        .containsOutput(
+            "Total number of buckets moved during this rebalance                                             | 0")
+        .containsOutput(
+            "Total time (in milliseconds) spent switching the primary state of buckets during this rebalance | 0")
+        .containsOutput(
+            "Total primaries transferred during this rebalance                                               | 0");
+
+    System.out.println("DEBR Finished rebalance two");
+  }
+
+  @Test
+  public void testSecondRebalanceIsNotNecessaryWithAddedAndRestartedMembers() {
+
+    addDataToRegion(NUMBER_OF_ENTRIES);
+
+    addOrRestartServers(2, 1);
+
+    gfsh.executeAndAssertThat("rebalance").doesNotContainOutput(
+        "Total bytes in all redundant bucket copies created during this rebalance                        | 0")
+        .doesNotContainOutput(
+            "Total time (in milliseconds) spent creating redundant bucket copies during this rebalance       | 0")
+        .doesNotContainOutput(
+            "Total number of redundant copies created during this rebalance                                  | 0")
+        .doesNotContainOutput(
+            "Total bytes in buckets moved during this rebalance                                              | 0")
+        .doesNotContainOutput(
+            "Total time (in milliseconds) spent moving buckets during this rebalance                         | 0")
+        .doesNotContainOutput(
+            "Total number of buckets moved during this rebalance                                             | 0")
+        .doesNotContainOutput(
+            "Total time (in milliseconds) spent switching the primary state of buckets during this rebalance | 0")
+        .doesNotContainOutput(
+            "Total primaries transferred during this rebalance                                               | 0");
+
+    gfsh.executeAndAssertThat("rebalance").containsOutput(
```

Comment:
Is it possible to extract the verification to it's own method for reuse?

Reply:
Done!